tox.ini

```diff
@@ -76,4 +76,3 @@ commands =
     pylint {[vars]all_path}
     isort --check-only --diff {[vars]all_path}
     black --check --diff {[vars]all_path}
-    mypy {[vars]all_path}
```

Comment:
Is that removal intentional and part of the PR?

Reply:
Yes, the mypy check are moved to a different story now.